src/Language/Haskell/Ghcid/Parser.hs

```diff
@@ -58,6 +58,11 @@ parseLoad  = nubOrd . f
         f [] = []
         isSpan c = c== ',' || c == '(' || c == ')'
 
+-- After the file location, message bodies are indented (perhaps prefixed by a line number)
+isMessageBody :: String -> Bool
+isMessageBody xs = isPrefixOf " " xs || isPipeIndented xs
+  where isPipeIndented = all (\x -> isSpace x || isDigit x) . takeWhile (/= '|')
```

Comment:
This doesn't require a pipe character - the line "12" matches for example. Should there be a requirement to find the pipe?